jest.config.js

```diff
@@ -1,5 +1,7 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
-  preset: 'ts-jest',
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'node'],
+  preset: 'ts-jest/presets/js-with-ts',
   testEnvironment: 'node',
+  transformIgnorePatterns: ["node_modules/(?!@probot/get-private-key)"],
```

Comment:
This had to be done because `@probot/get-private-key` doesn't distribute properly transformed JS. I'm open to better ways of doing this.